phpcs-ruleset.xml

```diff
@@ -27,6 +27,7 @@
     <!-- Exclude third party code. -->
     <exclude-pattern>profiles/common/libraries/</exclude-pattern>
     <exclude-pattern>profiles/common/modules/custom/ecas/libraries/</exclude-pattern>
+    <exclude-pattern>profiles/common/themes/europa</exclude-pattern>
```

Comment:
I saw that we have a `ec_resp` folder in the `profiles/common/themes`. We should exclude it from the CS too, no?